to_production.txt

```diff
@@ -1,5 +1,6 @@
 # New
 ofl/nationalpark # https://github.com/google/fonts/pull/9174
+ofl/playpensanshebrew # https://github.com/google/fonts/pull/9273
```

Comment:
I think that Viv wants to ship all of PPS i18n at once

Reply:
Reverted in https://github.com/google/fonts/pull/9309/commits/8a73d9789b3ea3ae3c87db75b2ef4fe9dcd6d671